### PR TITLE
✨ feat: Add hermes package

### PR DESCRIPTION
**Pull request by ruairidhwm**
# Description

Hermes allows you to easily send messages via: `slack`, `email`, and `sms` through a simple API.

For example:

  ```javascript
  await sendNotification({
      channels: ['email', 'slack', 'sms'],
      email: {
        to: user.email,
        subject: 'Reset your Password',
        template: TEMPLATE_RESET_PASSWORD,
        templateData: {
          link,
        },
      },
     slack: {
        channel: 'testing',
        message: 'User #1234 has requested a password reset.'
        emoji: ':fire:',
        blocks: [...block data],
      },
     sms: {
       to: user.phone,
       message: 'A password reset request was received for your account. If this was not you, please contact us immediately.'
     }
    })
  ```
# Changes

* ✨ feat: Add hermes package (f5bb649)

### CODE REVIEW

File: packages/hermes/src/email/mailer.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import nodemailer from "nodemailer";
+const mg = require("nodemailer-mailgun-transport");
```
Comment (elving) [inline packages/hermes/src/email/mailer.ts]: Why use `require` here?

File: packages/hermes/src/slack/sendSlack.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import { WebClient } from "@slack/web-api";
+
+const slack = new WebClient(process.env.SLACK_TOKEN);
```
Comment (elving) [inline packages/hermes/src/slack/sendSlack.ts]: Is there a specific reason why you are defining this variable before the next `import`? I think all `import` statement should be at the top of the file.

File: packages/hermes/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<p align="center">
+  <img width="200" height="200" src="https://i.imgur.com/BQ4A3bS.png">
```
[the remark this answers is not in the document]
Comment (ruairidhwm) [reply packages/hermes/README.md]: Hah, I wish! I designed it on Canva 😝

File: packages/hermes/README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+## Example
+
+```javascript
+await sendNotification({
+    channels: ['email', 'slack', 'sms'],
+    email: {
+      to: user.email,
+      subject: 'Reset your password',
+      template: TEMPLATE_RESET_PASSWORD,
```
[the remark this answers is not in the document]
Comment (ruairidhwm) [reply packages/hermes/README.md]: I'll add an example to the docs. It's a reference to the template names we use in Mailgun. For example it's `password-reset` in our own application which we store as a constant.

File: packages/hermes/src/email/mailer.ts (hunk 1 above)
Comment (ruairidhwm) [reply packages/hermes/src/email/mailer.ts]: Good point. Was a lazy copy and paste from the package docs.

File: packages/hermes/src/slack/sendSlack.ts (hunk 2 above)
Comment (ruairidhwm) [reply packages/hermes/src/slack/sendSlack.ts]: Agreed, that shouldn't be in there. Will edit.

**Comment by ruairidhwm** [comment]
> I think everything within `packages/hermes/.github` should be in the root of the repo since I don't think we can have workflows specific to certain packages (I may be wrong though).

No problem, I can shift that.

**Pull request merged**